test/registry/uninstall-plugin/uninstallPlugin.t.sol

```diff
@@ -21,10 +20,13 @@ contract UninstallPlugin_Test is Registry_Test {
     }
 
     function test_RevertWhen_PluginUnknown() external whenCallerHasProxy {
+        // Assert that every plugin method is uninstalled.
+        checkThatPluginMethodsArentInstalled(users.alice, plugins.basic);
+
```

Comment:
This pre-flight check is redundant because the test would *not* revert with the `PRBProxyRegistry_PluginUnknown` error if the basic plugin is installed (accidentally).